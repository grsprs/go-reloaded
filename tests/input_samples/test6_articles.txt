I saw a apple and a hour later a university.