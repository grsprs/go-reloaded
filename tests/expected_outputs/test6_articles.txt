I saw an apple and an hour later a university.